apps/pie-microsite/src/assets/styles/components/_tokensTable.scss

```diff
@@ -248,6 +248,18 @@ $tokensTable-item-vertical-padding: f.spacing(e);
     @include f.font-size('body-s');
 }
 
+.c-tokensTable-globalToken {
+    margin-top: f.spacing(b);
+
+    @include f.media('>=wide') {
+        margin-top: 0;
+
+        & .c-tokensTable-token {
+            margin-left: f.spacing(b);
+        }
+    }
+}
+
```

Comment:
@raoufswe could we chat about this? Not quite sure why this has been added as the previous styles for the global token used were signed off by jordi. Looking on the PR preview, the margin seems to be incorrect on this PR

<img width="540" alt="Screenshot 2023-02-07 at 08 47 55" src="https://user-images.githubusercontent.com/16766185/217196169-41d09764-e39f-4176-a758-b1252addfbd6.png">
<img width="670" alt="Screenshot 2023-02-07 at 08 47 46" src="https://user-images.githubusercontent.com/16766185/217196172-bb5c8751-0efa-41b0-980f-a2cdeeabbfad.png">